src/site/stages/build/process-cms-exports/validator.js

```diff
@@ -1,20 +1,25 @@
+const fs = require('fs');
+const path = require('path');
 const Ajv = require('ajv');
 
 const ajv = new Ajv();
 
-const {
-  entityReference,
-  entityReferenceArray,
-} = require('./schemas/common/entity-reference');
-const genericNestedString = require('./schemas/common/generic-nested-string');
-
 // Custom formats
 ajv.addFormat('custom-date-time', /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}/);
 
-// Common $ref schemas
-ajv.addSchema(entityReference, 'EntityReference');
-ajv.addSchema(entityReferenceArray, 'EntityReferenceArray');
-ajv.addSchema(genericNestedString, 'GenericNestedString');
+/**
+ * Loads all the schemas in a directory into ajv
+ */
+const loadSchemas = dir => {
```

Comment:
With this, we won't have to add schemas individually whenever we need them in a `$ref`.